src/frontend/App/hooks/useVerdierForBrev.ts

```diff
@@ -41,7 +81,22 @@ export const useVerdierForBrev = (
                 [EBehandlingFlettefelt.tomdatoRevurderingBT]: tilDato,
             }));
         }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [beløpsperioder]);
 
-    return { flettefeltStore };
+    return { flettefeltStore, valgfeltStore, delmalStore };
+};
+
+const harSamordningsfradrag = (beløpsperioder: IBeløpsperiode[]): boolean => {
+    return beløpsperioder.some(
+        (beløpsperiode) => beløpsperiode.beregningsgrunnlag.samordningsfradrag
+    );
+};
+
+const erOvergangstønad = (
+    beløpsperioder: IBeløpsperiode[] | IBeregningsperiodeBarnetilsyn[]
+): beløpsperioder is IBeløpsperiode[] => {
```

Comment:
returnerer denne `boolean`, kanskje?

Reply:
Den gjør det :-)
https://www.typescriptlang.org/docs/handbook/advanced-types.html#using-type-predicates

Reply:
Gjør det indirekte, men må være en typeguard for å slippe å caste typen.
https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards

Reply:
🤓 👏